resources/styles/components/task-step/reading/book-content/tips-for-success.less

```diff
@@ -1,19 +1,38 @@
 .tips-for-success {
-  margin: 20px 0;
-  padding: 20px 0;
-  #fonts > .sans(1.8rem, 2.8rem);
-  border-bottom: 1px solid @border-color;
 
   &:before {
-    display: inline-block;
-    content: "Tips for Success";
-    #fonts > .sans(3.2rem, 3.2rem);
-    font-weight: 900;
-    color: @tutor-secondary;
-    white-space: nowrap;
-    background: #232e66;
-    padding: 10px 20px;
-    margin: 0 500px 15px 0;
+    height: 35px;
+    content: attr(data-label);
+    background: @tutor-quaternary url("icon-light-bulb.svg");
```

Comment:
Does splitting this into `background-color: ` and `.tutor-reading-step-banner-icon(...)` not work?